examples/scripts/project/categorization/bootstrap_taxonomy.py

```diff
@@ -0,0 +1,169 @@
+"""Example script for bootstrapping a Tamr Categorization project with taxonomy categories"""
+import argparse
+import logging
+import pandas as pd
+import requests
+
+from typing import Dict, Any
+
+import tamr_toolbox as tbox
+from tamr_toolbox.project.schema_mapping import schema
+
+# Uncomment for bootstrapping categories at a specific taxonomy tier
+# from tamr_toolbox.project.categorization import metrics
+
+
+LOGGER = logging.getLogger(__name__)
+
+CATEGORY_ATTRIBUTE_NAME = "Category Name"
+FULL_PATH_LIST_NAME = "Full Path List"
+FULL_PATH_STR_NAME = "Full Path"
+COLUMN_NAMES = [FULL_PATH_STR_NAME, FULL_PATH_LIST_NAME]
+PK_NAME = "Primary Key"
+
+
+def main(*, instance_connection_info: Dict[str, Any], categorization_project_id: str) -> bool:
+    """
+    Bootstraps taxonomy in a categorization project
```

Comment:
Describe in detail what "bootstrapping" means in this context

Reply:
What do you think would be a good description? I am stuck... Please help!

Reply:
Maybe something like "Bootstraps the model for a categorization project by adding the taxonomy as a separate source with training labels"